Header header
geometry_msgs/Vector3 center
geometry_msgs/Vector3 rotation
float64 creation_time
float64 last_detection_time
string obj_class
float32 score

